### DISCUSSION TITLE
How can I download PDF files directly after clicking a link

### DISCUSSION THREAD

**Discussion by tahader02** [seleniumbase/SeleniumBase#3024] [Q&A] [answered]
I want to download the file directly using SB after clicking a link (so I don't want it to use the PDF viewer). Is there any way to pass these specific options to the driver like in this post https://stackoverflow.com/questions/43149534/selenium-webdriver-how-to-download-a-pdf-file-with-python

**Comment by mdmintz** [comment, marked as answer]
Set `external_pdf=True`.

https://github.com/seleniumbase/SeleniumBase/blob/538b5d162c313f842f85af80992de49dd4f44889/seleniumbase/plugins/sb_manager.py#L80

**Reply by tahader02** [reply]
Thanks for the quick reply and the great tool.
